recipes/newrelic/infrastructure/super-agent/logs/linux-logs.yml

```diff
@@ -6,9 +6,6 @@ displayName: Logs Integration
 description: New Relic install recipe for Logging in super-agent
 repository: https://github.com/newrelic/infrastructure-agent/tree/master/assets/examples/logging
 
-dependencies:
```

Comment:
@alvarocabanas Why are we removing the dependencies here?

Reply:
Because it's failing and saying incompatible. 
We should have this dependency in the future but we can leave it without for the moment.